kore/test/Test/Kore/Step/Simplification/AndTerms.hs

```diff
@@ -846,6 +848,98 @@ test_andTermsSimplification =
                     & Pattern.withCondition concrete
             actual <- simplifyUnify concrete symbolic
             assertEqual "" ([expect], [expect]) actual
+        , testCase "qq\\equals(false, X in []) = \\top" $ do
```

Comment:
Because this tests `simplifyEquals`, please move it to `test_equalsTermsSimplification`.